src/astUtils/reflection.ts

```diff
@@ -0,0 +1,193 @@
+    return statement?.constructor?.name === 'ExpressionStatement';
+}
+export function isCommentStatement(statement: Statement): statement is CommentStatement {
+    return statement?.constructor?.name === 'CommentStatement';
+}
+export function isExitForStatement(statement: Statement): statement is ExitForStatement {
+    return statement?.constructor?.name === 'ExitForStatement';
+}
+export function isExitWhileStatement(statement: Statement): statement is ExitWhileStatement {
+    return statement?.constructor?.name === 'ExitWhileStatement';
+}
+export function isFunctionStatement(statement: Statement): statement is FunctionStatement {
+    return statement?.constructor?.name === 'FunctionStatement';
+}
+export function isIfStatement(statement: Statement): statement is IfStatement {
+    return statement?.constructor?.name === 'IfStatement';
+}
+export function isIncrementStatement(statement: Statement): statement is IncrementStatement {
+    return statement?.constructor?.name === 'IncrementStatement';
+}
+export function isPrintStatement(statement: Statement): statement is PrintStatement {
+    return statement?.constructor?.name === 'PrintStatement';
+}
+export function isGotoStatement(statement: Statement): statement is GotoStatement {
+    return statement?.constructor?.name === 'GotoStatement';
+}
+export function isLabelStatement(statement: Statement): statement is LabelStatement {
+    return statement?.constructor?.name === 'LabelStatement';
+}
+export function isReturnStatement(statement: Statement): statement is ReturnStatement {
+    return statement?.constructor?.name === 'ReturnStatement';
+}
+export function isEndStatement(statement: Statement): statement is EndStatement {
+    return statement?.constructor?.name === 'EndStatement';
+}
+export function isStopStatement(statement: Statement): statement is StopStatement {
+    return statement?.constructor?.name === 'StopStatement';
+}
+export function isForStatement(statement: Statement): statement is ForStatement {
+    return statement?.constructor?.name === 'ForStatement';
+}
+export function isForEachStatement(statement: Statement): statement is ForEachStatement {
+    return statement?.constructor?.name === 'ForEachStatement';
+}
+export function isWhileStatement(statement: Statement): statement is WhileStatement {
+    return statement?.constructor?.name === 'WhileStatement';
+}
+export function isDottedSetStatement(statement: Statement): statement is DottedSetStatement {
+    return statement?.constructor?.name === 'DottedSetStatement';
+}
+export function isIndexedSetStatement(statement: Statement): statement is IndexedSetStatement {
+    return statement?.constructor?.name === 'IndexedSetStatement';
+}
+export function isLibraryStatement(statement: Statement): statement is LibraryStatement {
+    return statement?.constructor?.name === 'LibraryStatement';
+}
+export function isNamespaceStatement(statement: Statement): statement is NamespaceStatement {
```

Comment:
Looks like we're missing `isClassStatement`. Are there any other statements not represented here?

Reply:
Eh I took all the statements in `Statements.ts` and forgot the class statements which is in its own file :)